pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.580.1</version>
```

Comment:
The issue appears when using the plugin in Jenkins >= 1.576. Besides with the previous baseline the project does not build in Java 8 because of xstream issues.

For these reasons the baseline dump is proposed.